cg/meta/upload/mip/mip_rna.py

```diff
@@ -34,11 +34,12 @@ def upload(self, ctx: click.Context, case: Family, restart: bool) -> None:
         if DataDelivery.SCOUT in case.data_delivery:
             try:
                 ctx.invoke(upload_rna_to_scout, case_id=case.internal_id)
-                self.update_uploaded_at(analysis)
             except CalledProcessError as error:
                 LOG.error(error)
+                return
         else:
             LOG.warning(
                 f"There is nothing to upload to Scout for case {case.internal_id} and "
                 f"the specified data delivery ({case.data_delivery})"
```

Comment:
I'm not sure this should be a warning, or if it even needs to be logged at all 🤔